### DISCUSSION TITLE
From right here, right now, to galaxies far away ✨

### DISCUSSION THREAD

**Discussion by samus-aran** [community/community#38795] [Announcements]
Software has been a core driver in human progress over the past century—and that isn’t changing any time soon. At this very moment, software is powering the people and infrastructure behind all the societal, environmental, scientific, political, and cultural movements each and every one of us interact with on any given day. Stormy Peters (@stormypeters) will look at some of the people and projects that enable progress for today’s biggest humanitarian efforts—that dream up worlds our most beloved superheroes exist in, and that are looking at universes beyond our own.

## Join us for [Stormy Peters Day 2](https://watch.githubuniverse.com/live/dayTwoTrackOne) Keynote! 🗝️🗒️ and jump into the [conversation here](https://github.com/community/events/discussions) at GitHub Universe

<img width="1129" alt="Screenshot 2022-11-10 at 17 36 40" src="https://user-images.githubusercontent.com/4041275/201166929-6bd28104-4ba3-44f3-81ed-8c7509989d21.png">

**Comment by githubdlke** [comment]
firasdlke10@gmail.com

**Comment by ghost** [comment]
> Software has been a core driver in human progress over the past century—and that isn’t changing any time soon. At this very moment, software is powering the people and infrastructure behind all the societal, environmental, scientific, political, and cultural movements each and every one of us interact with on any given day. Stormy Peters (@stormypeters) will look at some of the people and projects that enable progress for today’s biggest humanitarian efforts—that dream up worlds our most beloved superheroes exist in, and that are looking at universes beyond our own.
> 
> 
> 
> ## Join us for [Stormy Peters Day 2](https://watch.githubuniverse.com/live/dayTwoTrackOne) Keynote! 🗝️🗒️ and jump into the [conversation here](https://github.com/community/events/discussions) at GitHub Universe
> 
> 
> 
> <img width="1129" alt="Screenshot 2022-11-10 at 17 36 40" src="https://user-images.githubusercontent.com/4041275/201166929-6bd28104-4ba3-44f3-81ed-8c7509989d21.png">
> 
>

**Comment by novium2** [comment]
Retiring this post as it is over 6 months old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
